corehq/apps/userreports/reports/specs.py

```diff
@@ -65,6 +65,12 @@ class ReportColumn(JsonObject):
     transform = DictProperty()
     calculate_total = BooleanProperty(default=False)
 
+    @classmethod
+    def wrap(cls, obj):
+        if 'display' not in obj:
+            obj['display'] = obj['column_id']
```

Comment:
do you need to check if `column_id` is in `obj`? I think if it's not that might raise a `KeyError` which then might not be caught and raised by the framework as a `BadSpecError` (which is desired since those are caught and shown to the user).

would also be nice to add test coverage for this. for example, [like this](https://github.com/dimagi/commcare-hq/pull/8814/files#diff-65c4a40cc04d6a20143b4b51b3a1f86dR88)